x-pack/plugins/secops/public/pages/home/index.tsx

```diff
@@ -132,7 +132,7 @@ const PageHeader = styled(EuiPageHeader)`
   background-color: ${props => props.theme.eui.euiColorLightestShade};
   position: fixed;
   width: calc(100% - 75px);
-  z-index: 1;
+  z-index: 10;
```

Comment:
Not with this PR but I can see a follow up where these numbers are put into a `const` type structure for ease of access and changes as needed.